app/controllers/agentAppointment/AppointNewAgentForms.scala

```diff
@@ -39,4 +39,12 @@ object AppointNewAgentForms {
   val manageMultipleProperties: Form[ManagePropertiesOptions] = Form(
     Forms.single("multipleProperties" -> EnumMapping(ManagePropertiesOptions, "error.multipleProperties.required")))
 
+  val submitAgentRelationship: Form[AgentRelationshipForm] =
```

Comment:
I dont see any test cases for this form.

Reply:
there are tests in AgentRelationshipControllerSpec